var/spack/repos/builtin/packages/py-isort/package.py

```diff
@@ -12,6 +12,7 @@ class PyIsort(PythonPackage):
     homepage = "https://github.com/timothycrosley/isort"
     url      = "https://pypi.io/packages/source/i/isort/isort-4.2.15.tar.gz"
 
+    version('4.3.20', sha256='c40744b6bc5162bbb39c1257fe298b7a393861d50978b565f3ccd9cb9de0182a')
     version('4.2.15', sha256='79f46172d3a4e2e53e7016e663cc7a8b538bec525c36675fcfd2767df30b3983')
 
     depends_on('py-setuptools', type='build')
```

Comment:
Missing dependencies in the latest version:
```python
      install_requires=[                                                        
          'futures; python_version < "3.2"',                                    
          'backports.functools_lru_cache; python_version < "3.2"',              
      ],                                                                        
      python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",           
```
You'll have to see when these dependencies were added.